doc/modules/ROOT/pages/projects.adoc

```diff
@@ -665,6 +665,17 @@ your project, you could customize it with the following:
 ((nil . ((projectile-project-name . "your-project-name-here"))))
 ----
 
+By default, compilation buffers are not writable, which allows you to
+e.g.  press `g` to restart the last command. Setting
+`projectile-comint-mode` to a non-nil value allows you to make
+projectile compilation buffers interactive, letting you e.g. test a
```

Comment:
I don't think "interactive" is the write term here.

Reply:
I used that word because it is the one that is used in the Emacs documentation of `compile` (`C-h f compile <ENTER>`): 

```
Additionally, with universal prefix arg, compilation buffer will be in
comint mode, i.e. interactive.
```

Reply:
Got it.